cpp/benchmarks/common/cudf_tpch_datagen/tpch_datagen.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/table/table.hpp>
+
+namespace CUDF_EXPORT cudf {
+namespace datagen {

```

Comment:
@robertmaynard @vyasr where should `CUDF_EXPORT` be added here? inner most or top most or all places?

Reply:
The current setup is correct.